### PR TITLE
[Merged by Bors] - feat(MeasureTheory): define `DomMulAct` action on `Lp`

### PR DESCRIPTION
**Pull request by urkud**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/MeasureTheory/Function/LpSpace/DomAct/Basic.lean [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+instance [NormedRing 𝕜] [Module 𝕜 E] [BoundedSMul 𝕜 E] : SMulCommClass Mᵈᵐᵃ 𝕜 (Lp E p μ) :=
+  Subtype.val_injective.smulCommClass (fun _ _ ↦ rfl) fun _ _ ↦ rfl
+
+instance [NormedRing 𝕜] [Module 𝕜 E] [BoundedSMul 𝕜 E] : SMulCommClass 𝕜 Mᵈᵐᵃ (Lp E p μ) :=
+  .symm _ _ _
+
+-- We don't have a typeclass for additive versions of the next few lemmas
+-- Should we add `AddDistribAddAction` with `to_additive` both from `MulDistribMulAction`
+-- and `DistribMulAction`?
```
Comment (eric-wieser) [inline Mathlib/MeasureTheory/Function/LpSpace/DomAct/Basic.lean:63]: Probably not unless we have any evidence that we would use it.

File: Mathlib/MeasureTheory/Function/LpSpace/DomAct/Basic.lean [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+Authors: Yury Kudryashov
+-/
+import Mathlib.MeasureTheory.Integral.Bochner
+import Mathlib.MeasureTheory.Function.AEEqFun.DomAct
+
+/-!
+# Action of `Mᵈᵐᵃ` on `Lᵖ` spaces
+
+In this file we define action of `Mᵈᵐᵃ` on `MeasureTheory.Lp E p μ`
+given by `mk c • ⟨⟨f, _⟩, _⟩ = ⟨⟨(f <| c • ·), _⟩, _⟩`.
```
Comment (ocfnash) [inline Mathlib/MeasureTheory/Function/LpSpace/DomAct/Basic.lean:13]: I think it is important to make the doc string human-friendly and I claim this line is not. How about:
```suggestion
If `f : α → E` is a function representing an equivalence class in `Lᵖ(α, E)`, `M` acts on `α`, and
`c : Mᵈᵐᵃ`, then `c • [f]` is represented by the function `a ↦ f (c • a)`.

```

**Review by ocfnash — APPROVED** [review]
Just two suggestions; I think the remark about the doc string is important.

Thanks!

bors d+

File: Mathlib/MeasureTheory/Function/LpSpace/DomAct/Basic.lean (hunk 1 above)
Comment (eric-wieser) [inline Mathlib/MeasureTheory/Function/LpSpace/DomAct/Basic.lean:63]: It looks to me like this comment has shifter to the wrong place. Can you put it right above the `DistribSmul`?

Comment (eric-wieser) [reply Mathlib/MeasureTheory/Function/LpSpace/DomAct/Basic.lean:63]: Oh, nevermind.

**Comment by urkud** [comment]
bors merge

**Pull request closed without merging**